package.json

```diff
@@ -16,8 +16,7 @@
     "ios"
   ],
   "dependencies": {
-    "react-native-vector-icons": ">=1.0.4",
```

Comment:
`react-native-vector-icons` should still be a dependency. Please do not remove it :)